plugin.go

```diff
@@ -95,11 +95,11 @@ func (p Plugin) Exec() error {
 	cmds = append(cmds, commandVersion())      // docker version
 	cmds = append(cmds, commandInfo())         // docker info
 	cmds = append(cmds, commandBuild(p.Build)) // docker build
-
 	for _, tag := range p.Build.Tags {
+		fmt.Fprintf(os.Stdout, "Tagging: %s\n", tag)
 		cmds = append(cmds, commandTag(p.Build, tag)) // docker tag
-
 		if p.Dryrun == false {
+			fmt.Fprintf(os.Stdout, "Pushing: %s\n", tag)
```

Comment:
this logging is un-necessary because we trace every command that gets executed with the trace function:

```
func trace(cmd *exec.Cmd) {
	fmt.Fprintf(os.Stdout, "+ %s\n", strings.Join(cmd.Args, " "))
}
```

as a result, you see this in your drone logs:

```
+ /usr/local/bin/docker tag 2530f8723f6c960d1ba389db4acaf5fe69d8550d drone/drone:0.5.0
```